### DISCUSSION TITLE
As proxy provider

### DISCUSSION THREAD

**Discussion by IvanIsak2000** [XTLS/Xray-core#3749] [Q&A] [answered]
I'm a noob in the proxy and vpn topic.

 YouTube was blocked in my country and I found a solution as xray. I'm just using custom xray config generator to generate it and after that run proxy. 
 
 I have idea to share my proxy server to my friends, but isolated than me. Cause I just want to generate new config file for xray and run a folder of my friend's configs. It is possible and normal to generate new config for each of my friends and simply restart/run server?

**Comment by meower1** [comment, marked as answer]
the best solution would be to use a management panel, there are many choices and are fairly simple to install and get used to. i personally suggest this one as its easy to setup and get used to

https://github.com/MHSanaei/3x-ui

after installing the panel create separate config for each friend

**Reply by IvanIsak2000** [reply]
Sounds cool! Thx
